tests/e2e/helper.ts

```diff
@@ -0,0 +1,22 @@
+import { Aptos, AptosConfig, Network, NetworkToFaucetAPI, NetworkToIndexerAPI, NetworkToNodeAPI } from "../../src";
+
+/**
+ * This function attempts to read env vars first to configure an Aptos client. If they
+ * aren't set, it falls back to the values for Network.LOCAL. Use this whenever you
+ * want a client to use with a local testnet.
```

Comment:
not completely understand "Use this whenever you want a client to use with a local testnet"? we should use this always and just change the network if we want to run against a different network

Reply:
There are places in the E2E tests which test against devnet / testnet. Though I think that's sus and shouldn't be the case.

Reply:
I believe we use it to test staking endpoints - which makes sense